Cairo Desktop/Cairo Desktop/Services/CairoApplicationThemeService.cs

```diff
@@ -33,10 +34,15 @@ public CairoApplicationThemeService(ILogger<CairoApplicationThemeService> logger
             _settings = settings;
 
             MigrateSettings();
-
+            SystemEvents.UserPreferenceChanged += SystemEvents_UserPreferenceChanged;
             _settings.PropertyChanged += Settings_PropertyChanged;
         }
 
+        private void SystemEvents_UserPreferenceChanged(object sender, UserPreferenceChangedEventArgs e)
+        {
+            SetThemeFromSettings();
```

Comment:
This is a bit expensive to do every time, could you add the following?
1. Only run if `e.Category` matches the appropriate category for accent color changes
2. Add a boolean resource to the theme that indicates if it uses system colors, and only run if that is true.

Reply:
Done, but it will be a marginal optimization, since e.Category does not handle theme colors and the enum value 'Color' was meant for SysColor changes, not the theme one.
I also put a boolean into the theme's resources:
Once SetTheme is called, it will merge the dictionary as before, but this time it will also scan the mergedDictionaries:
the listener will be subscribed or unsubscribed if the key is found in the dictionary, and its value is set to true.
Yet there's the problem... that changing the theme calls this too many times, maybe a debouncer could do.

Reply:
Hmm, that's unfortunate. Maybe using a registry change listener makes more sense, as we can be a lot more targeted.

Reply:
that will either require to install Microsoft.Windows.Compatibility nuget package, as net 6.0 does not support ManagementEventWatcher